### PR TITLE
Fix black color in search input

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

File: scss/misc/_search-input.scss [hunk 1]
```diff
@@ -29,3 +29,7 @@ input.search-input {
     width: calc(100% - 80px);
   }
 }
+
+input#chat-list-search {
+  color: #fff;
```
Comment (Simon-Laux) [inline scss/misc/_search-input.scss:34]: hardcoded white? can you use a theme variable?

**Comment by Simon-Laux** [comment]
can you rebase this?

File: scss/misc/_search-input.scss (hunk 1 above)
Comment (Jikstra) [reply scss/misc/_search-input.scss:34]: Didn't use a themevariable as currently also the X icon is a hardcoded white. But for sure, a theming variable makes sense here. But I have to admit that I hate adding theming variables :D

Comment (Simon-Laux) [reply scss/misc/_search-input.scss:34]: yeah, but I believe the var you are searching already exists: `navBarText`

**Comment by Simon-Laux** [comment]
after you re-based this we can merge this.

**Pull request merged**